### PR TITLE
New request: get the list of upcoming webinars.

### PR DESCRIPTION
**Pull request by Navega**
By Navega Software Developer at Appfluence Inc.

### CODE REVIEW

File: zoomus/components/webinar.py [hunk 1]
```diff
@@ -18,6 +18,12 @@ def list(self, **kwargs):
             kwargs['start_time'] = util.date_to_str(kwargs['start_time'])
         return self.post_request("/webinar/list", params=kwargs)
 
+    def upcoming(self, **kwargs):
```
Comment (prschmid) [inline zoomus/components/webinar.py]: Can you add a test file tests/zoomus/components/webinar/test_upcoming.py and add in the appropriate tests. Take a look at the other files there for examples. Thanks!

Comment (Navega) [inline zoomus/components/webinar.py]: Thanks! I added the file :)

Comment (Navega) [inline zoomus/components/webinar.py]: Thanks! This is ready :)

**Comment by Navega** [comment]
The test file was added to Pull Request.

**Comment by prschmid** [comment]
Looks like there is a failure on python 2.7 in one of your tests:

https://travis-ci.org/actmd/zoomus/jobs/238835235

**Comment by Navega** [comment]
Sorry... Could you check it now?

**Pull request merged**